pootle/apps/pootle_store/models.py

```diff
@@ -831,6 +831,15 @@ def update_qualitychecks(self, keep_false_positives=False,
     def get_qualitychecks(self):
         return self.qualitycheck_set.all()
 
+    def get_qualitychecks_critical(self):
+        return self.get_qualitychecks().filter(category=Category.CRITICAL)
+
+    def get_active_qualitychecks_critical(self):
```

Comment:
I think naming it `get_active_critical_qualitychecks` feels more natural. Similar to the previous method.